.circleci/workflows.yml

```diff
@@ -2740,6 +2764,12 @@ darwin-x64-workflow: &darwin-x64-workflow
         resource_class: macos.x86.medium.gen2
         requires:
           - darwin-x64-build
+    - server-unit-tests-cloud-environment:
```

Comment:
How come this job isn't run against `darwin-arm64` and `linux-arm64`?

Reply:
I suppose I could add it. I'm not sure there's much benefit as the logic is platform specific, not architecture specific